client/styles/admin.css

```diff
@@ -0,0 +1,74 @@
+/* SelectElement */
+.wfa-right-elements-list {
```

Comment:
Does this _expect_ the right side bar config to be in place?

Reply:
No, that's a custom class I add here:
https://github.com/nyeholt/silverstripe-workflow-actions/pull/1/files#diff-e4e683a478ef409169197f3975a28218R78